CODEOWNERS

```diff
@@ -234,6 +239,9 @@
 /lib/gui/                                 @vanwinkeljan
 /lib/os/                                  @andrewboie @andyross
 /lib/posix/                               @pfalcon
+/lib/cmsis_rtos_v2/                       @nashif
+/lib/cmsis_rtos_v1/                       @nashif
+/lib/libc/                                @nashif @andrewboie
 /lib/libc/minimal/source/stdlib/bsearch.c @balajikulkarni
```

Comment:
With /lib/libc/ added, I'd suggest to remove entry for /lib/libc/minimal/source/stdlib/bsearch.c .

Reply:
should be fine, @balajikulkarni will additionally be added as a reviewer here.